io.ts

```diff
@@ -28,7 +28,8 @@ export function createRequest(
   msg += `${command}\r\n`;
   for (const arg of _args) {
     const val = String(arg);
-    msg += `$${val.length}\r\n`;
+    const bytesLen = encoder.encode(val).length;
```

Comment:
[nits] byteLength is better?
```suggestion
    const bytesLen = encoder.encode(val).byteLength
```

Reply:
@manyuanrong Thank you for PR. Please check reviewed comments.